conda-recipe/run_test.bat

```diff
@@ -28,54 +28,45 @@ if "%PYTHON%"=="python" (
     set NO_DIST=1
 )
 
+
+
 %PYTHON% -c "from sklearnex import patch_sklearn; patch_sklearn()" || set exitcode=1
 
+set "PYTEST_ARGS= "
+
+IF DEFINED COVERAGE_RCFILE (set PYTEST_ARGS=--cov=onedal --cov=sklearnex --cov-config="%COVERAGE_RCFILE%" --cov-append --cov-report= %PYTEST_ARGS%)
```

Comment:
Actually on second thought, I think these double quotes around `%COVERAGE_RCFILE%` would have an unintended effect of leaving quotes in the argument, so perhaps it should either be done without the quotes, or with the quotes around the whole `set "var=value"` call.

Sorry for the confusion.

Reply:
No worries, have made each of the PYTEST_ARGS in quotes as you've suggested (on next commit)